lib/install/reconfigure/plan.go

```diff
@@ -91,7 +97,7 @@ func (p *Planner) GetOperationPlan(operator ops.Operator, cluster ops.Site, oper
 	builder.AddNodePhase(plan)
 	builder.AddDirectoriesPhase(plan)
 	builder.AddPodsPhase(plan)
-	builder.AddRestartPhase(plan)
+	builder.AddRestartPhase(plan, *runtimePackage)
```

Comment:
Nit: Why not make runtimePackage a part of the PlanBuilder struct? Teleport package is there and IIRC we do it the same way for other operations.